packages/picasso.js/src/core/chart-components/legend-cat/navigation-renderer.js

```diff
@@ -110,7 +110,8 @@ function render(renderer, {
     direction: order[0],
     attrs: {
       'data-action': 'next',
-      'data-component-key': legend.settings.key
+      'data-component-key': legend.settings.key,
+      tabindex: -1
```

Comment:
I would prefer if this was configurable to keep it accessible

Reply:
Fixed!